gamp-recipes/.env

```diff
@@ -1,2 +1,4 @@
 DATABASE_URL="mysql://root:123456@localhost:3306/gamp_db?schema=public"
```

Comment:
Lembre-se de criar um .env-example e um .gitignore no nível do gamp-recipes pra evitar expôr informações sensíveis de ambientes (seja de desenvolvimento, testes ou prod) pra não subir mais arquivos .env e deixar uma referência pra quem for desenvolver no repo